### ISSUE TITLE
Vue 2 Support

### ISSUE THREAD

**Issue by KidSysco** [lorisleiva/solana-wallets-vue#13]
Hello, 

Is there Vue 2 support for this?

**Comment by KidSysco** [comment]
The Vuetify framework does not yet support Vue 3. This is quite common with many large Vue frameworks.

If there is no Vue 2 support, do you think this would work from an iFrame if I built a vue 3 app just for this wallet?

I am also wondering if the solana components can just be used directly in a vue 2 project.

**Comment by KidSysco** [comment]
I tried the iFrame idea, and it did not work at all. I am not having much luck integrating with the @solana/wallet-adapter-base on my own.  But I am trying to mimic at your code as close as I can in Vue 2.

**Comment by KidSysco** [comment]
I think Vue 2 should be supported becuase there will be numerous apps still using vue 2 for at least a couple more years.

**Comment by ghulamMustafaRaza** [comment]
I agree with this proposal

**Comment by reconwhispers** [comment]
we agree with @KidSysco

**Comment by lorisleiva** [comment]
Hey 👋 as I said in that PR, I couldn't make it work but if anyone is able to, then I see no problem accepting contributions for it.

**Comment by KidSysco** [comment]
That is not really what you said.

"If this is a deal-breaker for many developer friends out there, reach out to me and I could support another version of this package tailored for Vue 2."

If what you are doing for Vue 3 breaks what you had already done for Vue 2, then doesn't it call for another package tailored for Vue 2?

No one is going to sign up for sometthing that you were unable to achieve. 

Giving up on Vue 2 like this is abandoning huge portions of the web for years to come.

If you will not lead this effort, I don't know who will.

**Comment by lorisleiva** [comment]
I'm disappointed with your answer to be honest. Open source is a place where we should be helping each other out to build better software not a place to whine when someone else is not able to do the work you'd benefit from.

You opened this issue with the goal of adding Vue 2 support on this package. I assumed you were ready to get your hands dirty to make that work and was willing to help you in the process.

To clarify, Vue 2 was never properly working on this package because Vue Demi does not work with component, just library logic. So I removed code that did not function after having tried to make it work for days pro-bono.

The truth is I don't use Vue 2 anymore so I have no incentive in spending more time creating and maintaining another library for it. If that's what you need and other developers need then there are two solutions: you can do it yourselves or we can help each other to reach a desired outcome.

In any case, complaining is not gonna get you far in open source.

**Comment by KidSysco** [comment]
I was rather disapointed with your answer too. You were the one who said you would tailor a package for Vue 2. You are the one who works for metaplex. All I did was repeat your own words back to you, and you call it "complaining". I have likely been in open source much longer than you, and I have made a great career for myself. So thanks for assuming that you know anything about me at all!

You just gave me the distinct feeling that you have no further interest in working on this project.

With that aside, I wonder if it is your technical vision that is not realistic here.

It seems to me that you are trying to create a Vue plugin that supports both Vue 2 and Vue 3, and I do not think that is going to be possible.
